inst/get-testdata/for-compute_importance-fn/exp_imp_mean_lasomo/mean-case1-no_missing-linear_pool.R

```diff
@@ -5,7 +5,7 @@
 # load the package to make its internal functions available
 devtools::load_all()
 source(system.file(
-  "get-testdata/for-score_untrained-fn/helper-exp_imp-untrained.R",
+  "get-testdata/for-score_untrained-fn/helper-exp_imp.R",
```

Comment:
All instances that reference this file need to be updated as follows to match the renaming of the `score_untrained` function 
```suggestion
  "get-testdata/for-compute_importance-fn/helper-exp_imp.R",
```